solutions/observability/applications/otel-rum.md

```diff
@@ -0,0 +1,967 @@
+---
+navigation_title: OpenTelemetry for Real User Monitoring (RUM)
+description: Instrument web applications with OpenTelemetry for Real User Monitoring using Elastic Observability.
+applies_to:
+  stack:
+  serverless:
+    observability:
+products:
+  - id: cloud-serverless
+  - id: observability
+---
+
+# OpenTelemetry for Real User Monitoring (RUM)
+
+:::{important}
+Using OpenTelemetry for Real User Monitoring (RUM) with {{product.observability}} is currently in **Technical Preview**. This feature is provided as-is and has [limitations](#known-limitations). It should not be used in production environments. Elastic provides best-effort support for Technical Preview features and they are not covered under standard SLAs.
+:::
+
+This documentation outlines the process for instrumenting your web application with OpenTelemetry browser instrumentation, using {{product.observability}} as the backend. Unlike the [EDOT SDKs](opentelemetry://reference/edot-sdks/index.md), this approach uses upstream OpenTelemetry JavaScript packages directly. The following sections detail the required components and their proper configuration to acquire traces, logs, and metrics from the application to visualize them within {{kib}}.
+
+While this guide uses upstream OpenTelemetry instrumentation, you can also use the [EDOT Collector](elastic-agent://reference/edot-collector/index.md) components as part of your data ingestion pipeline.
+
+## Prerequisites
+
+This guide assumes you're using an {{product.observability}} deployment. You can use an existing one or set up a new one. If you're new to {{product.observability}}, follow the guidelines in [Get started with {{product.observability}}](/solutions/observability/get-started.md).
+
+:::{warning}
+Avoid using OpenTelemetry alongside any other {{apm-agent}}, including Elastic {{product.apm}} agents. Running multiple agents in the same application process might lead to conflicting instrumentation, duplicate telemetry, or other unexpected behavior.
+:::
+
+### OTLP endpoint
+
+You have two main options for setting up an OTLP endpoint:
+
+1. Use an existing OpenTelemetry Collector: If you already have a Collector deployed in your infrastructure, you can configure it to accept telemetry from browsers. This requires configuring CORS headers either directly on the Collector (if it's publicly accessible) or through a reverse proxy.
+
+2. **Start from scratch with {{ecloud}}**: If you're setting up a new deployment, you can create an {{ecloud}} hosted deployment or {{serverless-short}} project, which includes the [{{motlp}}](opentelemetry://reference/motlp.md). This approach requires a reverse proxy to handle CORS configuration.
+
+#### Use an existing OpenTelemetry Collector [use-an-existing-open-telemetry-collector]
+
+If you already have an OpenTelemetry Collector in your infrastructure, you can reuse it to ingest the traces, metrics, and logs generated by your instrumented web applications.
```

Comment:
It's a bit unclear from this description (apart from the CORS config) where the "existing Collector" would send the data? 

- (a) i.e. would it at the same time serve as a gateway collector (i.e. aggregate trace metrics and send data directly to Elasticsearch using the ES exporter)? 
- (b) OR do we recommend that to be separate from a Gateway collector and it would send data through OTLP to another "Gateway collector" / mOTLP?

If it's (a) we should clearly state that it needs to be an EDOT-like collector (with all the required components included for metrics aggregation) with a corresponding config for metrics aggregation. If it's (b) we should state it explicitly as well.

IMHO, in the case of ECH and Serverless, we should rather recommend (b) as it simplifies things.

Reply:
+CC @hegerchr @david-luna